src/Core/Domain/Cart/Command/UpdateCartDeliverySettingsCommand.php

```diff
@@ -43,14 +40,28 @@ class SetFreeShippingToCartCommand
      */
     private $allowFreeShipping;
 
+    /**
+     * @var bool
+     */
+    private $isAGift;
+
+    /**
+     * @var bool
+     */
+    private $useRecycledPackaging;
+
     /**
      * @param int $cartId
      * @param bool $allowFreeShipping
+     * @param bool $isAGift
+     * @param bool $useRecycledPackaging
      */
-    public function __construct(int $cartId, bool $allowFreeShipping)
+    public function __construct(int $cartId, bool $allowFreeShipping, ?bool $isAGift, ?bool $useRecycledPackaging)
```

Comment:
```suggestion
    public function __construct(int $cartId, bool $allowFreeShipping, ?bool $isAGift = null, ?bool $useRecycledPackaging = null)
```

Maybe they can be optional? (even if in the controller you always set them which is a good choice, other devs won't have to for `null` parameter if they don't use it)

Reply:
Oh and seems like that's the reason why travis tests fail, the constructor expects 4 parameters instead of the 2 provided

Reply:
Yeah I rushed this part 😞 obviously Travis punishes me